utils/docker/docker.go

```diff
@@ -0,0 +1,33 @@
+package docker
```

Comment:
Can we put this in the `container/docker` package instead? I was trying to consolidate some of the docker logic in https://github.com/google/cadvisor/pull/1221/files